test_scripts/API/PerformAudioPassThru/034_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_MandatoryParams_STATIC.lua

```diff
@@ -0,0 +1,106 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] "audioPassThruIcon" does NOT exist at 'AppStorageFolder'
+-- [HMI API] UI.PerformAudioPassThru request/response
+-- [HMI API] TTS.Speak request/response
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case mobile app sends PerformAudioPassThru_request to SDL 
+-- with valid <audioPassThruIcon> parameter
+-- and the requested <audioPassThruIcon> does NOT exist at app`s sandbox
+-- and with another related to request valid params
+-- transfer UI.PerformAudioPassThru (<audioPassThruIcon>, other params)_request + Speak_request (depends on parameters provided by the app) to HMI
+--respond with <resultCode_received_from_HMI> + <info> to mobile app (expected resultCode from HMI is "WARNINGS" and info: "Reference image(s) not found" 
+-- as described in [F-S] Conditions for SDL to handle case when requested <Image> does not exist at "AppStorageFolder"
+--
+--1. Used preconditions
+--1.1. audioPassThruIcon does not exist in created by app AppStorageFolder
+--1.2. Request is sent with valid audioPassThruIcon parameter with type STATIC and with all mandatory parameters within bounds (samplingRate, maxDuration, bitsPerSample, audioType)
+--2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, mandatory params) from mobile to SDL and check:
+--2.1 SDL sends UI.PerformAudioPassThru (audioPassThruIcon, mandatory params) to HMI
```

Comment:
@maria-mih, points 2.1 / 2.2 are not steps, they are expectations of the step "Send..." and should be separate, something like:

    Send PerformAudioPassThru (audioPassThruIcon, other params) from mobile to SDL and check:
    SDL sends UI.PerformAudioPassThru
    SDL sends TTS.Speak to HMI
    HMI sends UI.PerformAudioPassThru (WARNINGS) to SDL